fundamentals/examples.tex

```diff
@@ -151,79 +151,111 @@ \subsection{Simulations}
-  </BatchReactor>
-  ...
-\end{lstlisting}
+This section will discuss the creation and results of a few simple fuel cycle
+simulations to help illustrate the flexibility of \Cyclus. These simulations
+are designed to illustrate Cyclus' capability.  Many simplifying assumptions
+have been made with respect to material compositions, fuel transmutation, and
+others. The three fuel cycles examined are:
+
+\begin{enumerate}
+    \item Once through (no recycle) 
+    \item Modified open (1-pass recycle)
+    \item closed (infinite-pass recycle)
+\end{enumerate}
+
+For each of these fuel cycles, a 1,100 month \Cyclus simulation with only a
+single reactor is run in addition to a simulation with 10 reactors with
+staggered refueling times.  As the number of staggered-cycle reactors
+increases the system converges toward continuous material flow results.
+\Cyclus flexibility allows this transition to be examined.  The facilities
+used in these simulations are:
+
+\begin{itemize}
+
+    \item \textbf{Reactor} (\Class{cycamore::Reactor}): This is a reactor
+        model that requests any number of input fuel types and uses static
+        compositions to transmute them to after they are burnt and discharged
+        from the core.  Due to the static spent fuel compositions spent fuel
+        compositions become "incorrect" as the input fuel diverges from the
+        fixed, expected input composition. The reactor was configured to model
+        a light water reactor with a 3 batch core (20,000 kg HM per batch)
+        operating on an 18 month cycle with a 2 month refuel time.  The
+        reactor was also configured to take in either enriched UOX fuel or
+        recycled MOX fuel - preferring the latter if available.
+
+    \item \textbf{Recycled Fuel Fabrication} (\Class{cycamore::Fab}): This
+        facility requests depleted uranium and separated fissile material and
+        mixes them together into recycled MOX fuel that it offers to
+        requesters.  The streams are mixed using a D-factors approach
+        \TODO{add D-factors citation} in an attempt to match simple neutronics
+        properties of an ideal target fuel composition.
+
+    \item \textbf{Separations} (\Class{cycamore::Separations}): This facility
+        takes in all kinds of spent fuel and separates it into plutonium and
+        waste streams with some efficiency (0.99 was used for these
+        simulations).  Up to 60,000 kg of Pu can be separated per month.
+
+    \item \textbf{Repository} (\Class{cycamore::Sink}): This is an
+        infinite-capacity facility that can take in all types of material
+        including separations waste streams and spent reactor fuel.
+
+    \item \textbf{UOX Fabrication} (\Class{cycamore::Source}): This facility
+        provides enriched UOX fuel as requested with infinite capacity.
+
+    \item \textbf{DU Source} (\Class{cycamore::Source}): This facility provides
+        depleted uranium as requested with infinite capacity.
```

Comment:
Same comment as above. "as requested with infinite capacity" is confusing. Suggestion: "as requested. This facility has infinite production capacity."